libs/client/ui/components/src/lib/map/map.tsx

```diff
@@ -0,0 +1,101 @@
+import { Park } from '@bregenz-bewegt/client/types';
+import { useEffect, useState } from 'react';
+import './map.scss';
+import {
+  Circle,
+  MapContainer,
+  Marker,
+  Popup,
+  TileLayer,
+  useMap,
+  useMapEvents,
+} from 'react-leaflet';
+import { Loading } from '@bregenz-bewegt/client-ui-pages';
+import { closeCircleOutline } from 'ionicons/icons';
+import { IonButton, useIonToast } from '@ionic/react';
+import { LocationEvent } from 'leaflet';
+import { Gps, GpsSlash } from 'iconsax-react';
+
+export interface MapProps {
+  parks: Park[];
+}
+
+export const Map: React.FC<MapProps> = ({ parks }: MapProps) => {
```

Comment:
Frage: Findesch es besser wenns die liste und die map in da components sin? Vllt künnt mas so wie mas bei bam profile gmacht hon maha. dass mas im pages folder atomisiert:

```
pages/
├─ src/
│  ├─ lib/
│  │  ├─ start/
│  │  │  ├─ map/
│  │  │  │  ├─ map.tsx
│  │  │  │  ├─ map.scss
│  │  │  ├─ list/
│  │  │  │  ├─ list.tsx
│  │  │  │  ├─ list.scss
│  │  │  ├─ start.tsx
│  │  │  ├─ start.scss
```